src/Shortcode/ShortcodeInterface.php

```diff
@@ -34,7 +34,7 @@ public function getParameters();
      * without value
      *
      * @param string $name Parameter name
-     * @param null $default Value returned if there is no parameter with given name
+     * @param mixed $default Value returned if there is no parameter with given name
```

Comment:
@funkjedi Thanks for your PR, but I have one question - shouldn't it be `string|null` instead of catch-all `mixed`? Either the parameter exists (and it'll be a string), or not (and it'll be null). Of course, the implementations could put anything in the class, but when you are using the library, the parameter value extracted from the text will be constrained to be either text or `null`. What do you think?

Reply:
Makes sense.  If that's the case the `@return` should probably also be changed to `string|null` instead of `mixed` as well.

Reply:
Yes, both types should be `string|null`. Can you update the PR? I'll merge it and release a new version of Shortcode soon.